RecoTracker/MkFit/plugins/MkFitEventOfHitsProducer.cc

```diff
@@ -54,13 +55,15 @@ MkFitEventOfHitsProducer::MkFitEventOfHitsProducer(edm::ParameterSet const& iCon
       mkFitGeomToken_{esConsumes()},
       qualityToken_{esConsumes()},
       geomToken_{esConsumes()},
-      putToken_{produces<MkFitEventOfHits>()} {}
+      putToken_{produces<MkFitEventOfHits>()},
+      useDeadModules_{iConfig.getParameter<bool>("useDeadModules")} {}
 
 void MkFitEventOfHitsProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
 
   desc.add("pixelHits", edm::InputTag{"mkFitSiPixelHits"});
   desc.add("stripHits", edm::InputTag{"mkFitSiStripHits"});
+  desc.add("useDeadModules", false)->setComment("Use dead modules information");
```

Comment:
`MeasurementTracker` (which plays somewhat similar role to `mkfit::EventOfHits` appears to use "bad" nomenclature
https://github.com/trackreco/cmssw/blob/601fbc54e314860dd9f9fcae3569794388e3c306/RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc#L135-L139
and specifically `UseStripStripQualityDB`
https://github.com/trackreco/cmssw/blob/601fbc54e314860dd9f9fcae3569794388e3c306/RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc#L91-L96
to control whether the `SiStripQuality` is used or not. I'd suggest to use same or similar nomenclature here.

Reply:
'useDeadModules' has been replaced with 'useStripStripQualityDB', for consistency with existing nomenclature.